### DISCUSSION TITLE
Legal: maybe a 'is it legal' / 'how do i comply to what license

### DISCUSSION THREAD

**Discussion by threader** [community/community#138982] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Wops. I wrote it all in the title. Im about to sleep and have been taking notes all day, you can check my github and please get back to me of you spot something i should change?

**Comment by LiteBrite82** [comment]
Hi @threader ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by threader** [reply]
Aha, no stress. I think i got my point well across on that whim of a thought. An actual ' legal AI ' is really a good idea and ' might save a devs. life ' and i hope you can see it trough.

**Comment by threader** [comment]
Omfg, what about the Longhorn sources? I remember trying a beta, woah
